git/options_test.go

```diff
@@ -186,6 +186,7 @@ func TestAuthOptionsFromData(t *testing.T) {
 			data: map[string][]byte{
 				"username":    []byte("example"), // This takes precedence over the one from the URL
 				"password":    []byte("secret"),
+				"bearerToken": []byte("token"),
```

Comment:
```suggestion
				"bearerToken": []byte("bearer_token"),
```

Reply:
Thanks for looking at my PR @aryan9600!

I think perhaps you meant to suggest this?`token` is the test value.

```suggestion
				"bearer_token": []byte("token"),
```

Reply:
yes, sorry my bad 🤦

Reply:
Okey, I will setup my test again with these changes to make sure it works as expected.

Reply:
I'm finished with the practical test, and flux was able to connect to bitbucket with a secret configured like this:

```
data:
  bearer_token: abc...
```

A commit with the changes has been pushed.